### PR TITLE
Dmirano/4328 speed up apd data test

### LINKED ISSUE #4328: [Cypress] Speed up apd-data test
### Description and related issues
The APD data test is already testing a lot. We need to move the delete test from apd-data to apd-basic (which is already covering a lot. The APD data test should just test adding data and validating that it was added correctly and displays correctly everywhere (including budget values).

We should also remove all of the places where the tests are doing math and replace it with a fixture that has whatever the static value should be. For example in the HIE test:
```
const staffTotal = staff1.costs[i] * staff1.ftes[i] + staff2.costs[i] * staff2.ftes[i];
const expenseTotal = expense1.costs[i] + expense2.costs[i];
const contractorTotal = contractor1.FFYcosts[i] + contractor2.FFYcosts[i][0] * contractor2.FFYcosts[i][1];
const activityTotalCosts = staffTotal + expenseTotal + contractorTotal;
const otherFunding = activityData.costAllocation.costs[i];
```
staffTotal, expenseTotal contractorTotal, activityTotalCosts and otherFunding should just be values that we pull from the HIE fixture.

### This task is done when…

- [x] all of the delete tests have been moved from apd-data to apd-basic
- [x] the math has been removed from the apd-data test

### PR DESCRIPTION
**Pull request by mirano-darren**
Resolves #4328 

### Description
Speeds up 03-apd-with-data test by not testing delete any more with activity 1, and not doing any math calculations in activity 2 and 3. Also had to update 04-budget-checks because of this change. 

### Significant changes or possible side effects
Nothing outside of Cypress

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |

None written, just updated existing tests (03-apd-with-data and 04-budget-checks)

### Steps to manually verify this change

1. Verify Cypress test pass on Circle

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] Associated OpenAPI documentation has been updated
- [x] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [x] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: e2e/cypress/integration/02-apd/02-default-apd.cy.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ describe('Default APD', { tags: ['@apd', '@default', '@slow'] }, () => {
       testDefaultActivityScheduleSummary(years);
     });
 
-    describe('default Proposed Budget', () => {
+    describe.only('default Proposed Budget', () => {
```
Comment (thetif) [inline e2e/cypress/integration/02-apd/02-default-apd.cy.js:77]: I think you want to remove only

File: e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js [hunk 2]
```diff
@@ -193,15 +193,15 @@ describe('APD with Data', { tags: ['@apd', '@data', '@slow'] }, () => {
       cy.findAllByText('State Staff and Expenses').eq(0).click();
 
       cy.fixture('HIT-activity-template.json').then(data => {
-        fillOutActivityPage.fillStateStaff(years, data.staff, false, true);
+        fillOutActivityPage.fillStateStaff(years, data.staff, false, true); // DELETE THESE BOOLEANS
```
Comment (thetif) [inline e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js:196]: is this a to-do for later?

File: e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js [hunk 3]
```diff
@@ -352,12 +352,12 @@ describe('APD with Data', { tags: ['@apd', '@data', '@slow'] }, () => {
...
         years,
         budgetData.afterStateExpenseNoFFP,
-        [553250, 557750],
+        [1173674, 1311620],
         'noFFP'
       );
     });
 
-    it('Checks Private Contractors Budget', () => {
+    it.skip('Checks Private Contractors Budget', () => {
```
Comment (thetif) [inline e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js:360]: do we need to still skip this?

**Review by thetif — APPROVED** [review]
looks good, love seeing all that math go

**Review by tbolt — APPROVED** [review]
Looks good, agree with Tifs comments

File: e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js (hunk 2 above)
Comment (mirano-darren) [reply e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js]: fixed!

File: e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js (hunk 3 above)
Comment (mirano-darren) [reply e2e/cypress/integration/02-apd/04-apd-budget-checks.cy.js]: still had to do this section at the time. but with my most recent commit I added it back in

File: e2e/cypress.config.js [hunk 4]
```diff
@@ -13,6 +13,7 @@ module.exports = defineConfig({
     viewportHeight: 1000,
     defaultCommandTimeout: 36000,
     videoUploadOnPasses: false,
+    numTestsKeptInMemory: 5,
```
[the remark this answers is not in the document]
Comment (mirano-darren) [reply e2e/cypress.config.js:16]: I think we could do that, @thetif is that possible?

**Pull request merged**